lib/pubid_iso/urn.rb

```diff
@@ -0,0 +1,49 @@
+module PubidIso
+  class URN
+    attr_accessor :identifier
+
+    def initialize(identifier)
+      @identifier = identifier
+    end
+
+    def to_s
+      result = "urn:iso:std:#{publisher}:#{identifier.number}"
+
+      if identifier.part
+        result += ":-#{identifier.part}"
+      end
+
+      if identifier.stage
+        result += ":stage-#{stage}"
+      end
+      result
+    end
+
+    def stage
+      # Stage 10: NP (non-public)
+      # Stage 20: WD (non-public)
+      # Stage 30: CD
+      # Stage 40: DIS
+      # Stage 50: FDIS
+      # Stage 50.60: PRF ("proof") (non-public)
```

Comment:
I believe PRF is stage 60.00. The published document is 60.60. 50.60 means the published FDIS. (substage 60 = published).